core/domain/classifier_services.py

```diff
@@ -113,7 +113,10 @@ def verify_signature(
 def handle_trainable_states(
     exploration: exp_domain.Exploration,
     state_names: List[str]
-) -> None:
+) -> List[
+    classifier_models.StateTrainingJobsMappingModel |
+    classifier_models.ClassifierTrainingJobModel
```

Comment:
cc @sahiljoster32 It is possible to use `|` for unions (see https://mypy.readthedocs.io/en/stable/runtime_troubles.html#using-x-y-syntax-for-unions), do we care?

Reply:
Yeah, we can use `|` but I think we should prefer to use `Union`. Because Unions are backward compatible and `|` is introduced in only `3.10` ( Refrence: In your mentioned link, It was mentioned in the first line `Starting with Python 3.10 ...` ). And Some new contributors might be confused by this annotation because `|` is also used in jobs and also used as OR operator ( not in oppia but in python general ).